photon-core/src/main/java/org/photonvision/vision/pipeline/ArucoPipelineSettings.java

```diff
@@ -47,7 +48,7 @@ public ArucoPipelineSettings() {
         pipelineType = PipelineType.Aruco;
         outputShowMultipleTargets = true;
         targetModel = TargetModel.kAprilTag6p5in_36h11;
-        cameraExposureRaw = -1;
+        cameraExposureRaw = 20;
```

Comment:
Changes from -1 to 20 for some reason setting -1 causes the camera to not use auto exposure.

Reply:
Now in-line with apriltag too